src/commands/dev/lib/dev-builder.ts

```diff
@@ -74,7 +72,7 @@ async function executeBuilds(
           config: build.config || {},
           isDev: true
         });
-        results = { ...results, ...output };
+        Object.assign(results, output);
```

Comment:
Why prefer `Object.assign` over object spread? 🤔

Reply:
It's a very minor optimization, but in this case the `results` object is not being re-created upon each pass of the loop.